src/core/Plugin.php

```diff
@@ -1382,6 +1384,21 @@ public function enqueue_scripts($hook_suffix)
 
         $legacyPlugin = Factory::getLegacyPlugin();
 
+        $term_author_slug = '';
+
+        if (
+            is_admin()
+            && $pagenow === 'term.php'
+            && isset($_GET['taxonomy']) && $_GET['taxonomy'] === 'author'
+            && isset($_GET['tag_ID'])
+        ) {
+            $author = Author::get_by_term_id((int)$_GET['tag_ID']);
+
+            if (is_object($author) && !is_wp_error($author) && isset($author->link)) {
+                $term_author_slug = $author->link;
```

Comment:
The name `$term_author_slug` do not reflect the data the variable is storing. It stores the link, not the slug.

Reply:
Thanks @andergmartins, the PR is now updated.